expenser-server/main.go

```
@@ -0,0 +1,71 @@
+	"os"
+
+	"github.com/joho/godotenv"
+	"github.com/keertirajmalik/expenser/expenser-server/db"
+	"github.com/keertirajmalik/expenser/expenser-server/handler"
+	"github.com/keertirajmalik/expenser/expenser-server/middleware"
+	"github.com/keertirajmalik/expenser/expenser-server/model"
+	_ "github.com/lib/pq"
+)
+
+func main() {
+	godotenv.Load()
```

golangci-lint

Error return value of `godotenv.Load` is not checked

(errcheck)